WpfApp/View/MainWindow.xaml

```diff
@@ -30,12 +30,19 @@
             <DataTemplate DataType="{x:Type vm:DataVisualiserViewModel}">
                 <local:DataVisualiser/>
             </DataTemplate>
+            <DataTemplate DataType="{x:Type vm:SettingsViewModel}">
+                <local:SettingsView/>
+            </DataTemplate>
         </Grid.Resources>
         <ContentControl Content="{Binding CurrentView}" Grid.Row="1" />
 
         <StackPanel Orientation="Horizontal" Margin="10">
             <Button Content="{materialDesign:PackIcon Kind=Home}" Command="{Binding NavigateToHome}" ToolTip="Home" Margin="0,0,5,0"/>
             <Button Content="{materialDesign:PackIcon Kind=DataUsage}" Command="{Binding NavigateToVisualiser}" ToolTip="Visualizer"/>
         </StackPanel>
+
+        <StackPanel Orientation="Horizontal" Margin="10" HorizontalAlignment="Right">
+            <Button Content="{materialDesign:PackIcon Kind=Settings}" Command="{Binding NavigateToSettings}" ToolTip="Home" Margin="0,0,5,0"/>
```

Comment:
@OwenCross289 ToolTip is incorrect on this settings button. I have changed it for you.